mods/default/nodes.lua

```diff
@@ -1613,6 +1617,96 @@ minetest.register_node("default:bush_sapling", {
 	end,
 })
 
+minetest.register_node("default:huckleberry_bush_stem", {
+	description = "Huckleberry Bush Stem",
```

Comment:
Yes best use the generic stem.

Reply:
Is the stem recoloured? If it needs to be a different colour to the generic stem to be realistic then a dedicated stem is fine.

Reply:
It's recoloured, but unlike all the other bush stems doesn't have any wood to be crafted into. Bush stems themselves are not great decorative items either, so it effectively makes this node pointless.

Maybe now the spawn mod sets default starting biomes we don't need bush stems to give wood, and could change them to giving sticks? That would give this new bush stem a crafting progression and seems more realistic (planks of wood from bush stems is quite silly after all).

Reply:
Yeah i'm ok with using the generic stem for this, fairly neutral.

Reply:
I'm happy to switch either to generic or use the new stem with the understanding we'll move to crafting stems to sticks (that could be a later PR though, and by one of us, so that it doesn't delay this).